meetings/2024-02-28.md

```diff
@@ -0,0 +1,141 @@
+
+---
+
+## Topics
+
+### NDNComm panel announcement
+URL: https://www.nist.gov/news-events/events/ndncomm2024
+
+>RG: Prof Lixia Zhang (UCLA) is organizing NDNComm at [NIST](https://www.nist.gov) with a panel titled "From local-first to fully-decentralized applications"
+>RG: Description: There exist many ongoing efforts aiming to steer Internet applications towards decentralized realizations; one such work was presented at last IETF "Local-first software, resilient and secure collaboration".  This panel will showcase examples of decentralized designs; articulate their commonalities and differences; share experience from the different efforts; and identify remaining challenges in building secure, resilient, and easy to use and deploy codebase with well defined API to facilitate new generations of decentralized apps.
+>RG: Would anyone in this meeting be interested in, or know someone who would be interested in, speaking about Solid and, especially, Linked Data, at this Panel? Please get in touch with me over chat. 
+>RG: Everyone is invited to join as participants.
+
+
+### Demo on indexing within Solid
+Proposed by Maxime Lecoq (INRIA/Startin'Blox)
+
+* ML: We will show an integration of indexes querying into the Startin'Blox framework over a federation of 32 live Hubl instances.
+* eP: Where is the original source, and where is it indexed? Only for open public data, or takes access control into account?
+* ML: Public but plan to support authenticated data.
+* ML: Generated by federation server. In the future, the data is modified on an instance, and notifies to federation to regenerate.
+* eP: Could it be done using a notification channel?
+* ML: Yes.
+* eP: Requirements on the source of the data?
+* ML: Profiles of users?
+* eP: Yes.
+* ML: It's RDF. There are not WebID profiles yet.
+* eP: How does adding to the index work?
+* ML: We index everyone. In the future, don't know. Need to consult. Perhaps consent.
+* MdJ: Can you summarise the conclusion on what practitioners need from specs?
+* ML: See video from the [presentation on Solid practitioners](https://github.com/solid-contrib/practitioners/blob/main/meetings/2024-02-19.md). Federated or distributed. 
+* MdJ: Do you think we need to add this to the Solid spec?
+* ML: I don't know. We are in the client-client domain. TypeIndex type of thing, which is not in the Protocol. It can be an addition.
+* MdJ: If you want different servers together, then they need to communicate.
+* ML: In the client-client standard we say we use this kind of index.
+* TBL: Data serialization of client-client should be in different spec.
+* eP: How do you <abbr title="Follow Your Nose">FYN</abbr>?
+* ML: We're making SPARQL queries with Comunica. Fetch metadata index, then know where to go for, e.g., skills, cities. Then from the skill index, then continue to find the appropriate index with defined `ex:forProperty`, `ex:forValue`. Trying to make generic vocab for indexing. And also `rdfs:seeAlso`.
```

Comment:
```suggestion
* ML: We're making SPARQL queries with Communica. Fetch metadata index, then know where to go for, e.g., skills, cities. Then from the skill index, then continue to find the appropriate index with defined `ex:forProperty`, `ex:forValue`. Trying to make generic vocab for indexing. And also `rdfs:seeAlso`.
```

Reply:
It's actually spelt like that: https://comunica.dev/about/

Reply:
It's worth linking.
```suggestion
* ML: We're making SPARQL queries with [Comunica](https://comunica.dev/about/). Fetch metadata index, then know where to go for, e.g., skills, cities. Then from the skill index, then continue to find the appropriate index with defined `ex:forProperty`, `ex:forValue`. Trying to make generic vocab for indexing. And also `rdfs:seeAlso`.
```